### PR TITLE
[15.0][IMP] account_commission: skip not invoice lines

### PR DESCRIPTION
**Pull request by cormaza**
- Modified original module account_commission to make filter on applicable lines heritable and skip not invoice lines

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Wouldn't be enough to only filter those lines with `exclude_from_invoice_tab=False` for removing also anglo-saxon lines?

**Comment by cormaza** [comment]
> Wouldn't be enough to only filter those lines with `exclude_from_invoice_tab=False` for removing also anglo-saxon lines?

You're right, but I think it's also a good idea to make it an independent function that returns the lines, to be able to customize, what do you think?

**Comment by pedrobaeza** [comment]
OK, extract it to the hook with such filter, and your module is not needed at all.

**Comment by cormaza** [comment]
> OK, extract it to the hook with such filter, and your module is not needed at all.

What do you suggest to continue in this pr or open another?

**Comment by pedrobaeza** [comment]
What is easier for you. If continuing in this PR, rename the commit and PR title.

File: account_commission/models/account_move.py [hunk 1]
```diff
@@ -136,6 +136,13 @@ def _compute_any_settled(self):
         for record in self:
             record.any_settled = any(record.mapped("agent_ids.settled"))
 
+    def filter_commission_applicable_lines(self):
```
Comment (pedrobaeza) [inline account_commission/models/account_move.py]: Make it private, as it's not called from RPC:

```suggestion
    def _filter_commission_applicable_lines(self):
```

Comment (cormaza) [reply account_commission/models/account_move.py]: Done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's clean the pending PRs:

/ocabot merge patch

Please fw-port it to v16

**Pull request merged**